src/server/common.cc

```diff
@@ -25,6 +27,8 @@ extern "C" {
 #include "server/server_state.h"
 #include "server/transaction.h"
 
+ABSL_FLAG(uint32_t, interpreter_per_thread, 5, "Lua interpreters per thread");
+
```

Comment:
its strange common.cc is responsible for initializing server state 🤨

Reply:
I agree. pull it out in another pr

Reply:
We can also increase it to about 10-15, above it usually delivers only small gains (given that most scripts are most likely a few ops), but I'm not sure why to keep the values low, it the interpreter overhead is likely super small

Reply:
Lets make it 10